examples/fluid/dam_break_2d_surface_tension.jl

```diff
@@ -0,0 +1,38 @@
+# This example shows how surface tension can be applied to existing cases
+# and which parameters have to be changed!
+using TrixiParticles
+
+fluid_density = 1000.0
+
+H = 0.6
+fluid_particle_spacing = H / 60
+
+# Set the surface tension to a value that is accurate in your case.
+# Note: This usually requires calibration to be physically accurate!
+surface_tension = SurfaceTensionAkinci(surface_tension_coefficient=0.25)
+
+# `density_diffusion` is deactivated since the interaction with the surface tension model can
+# cause stability problems.
+# `adhesion_coefficient` needs to be set to a value so that the fluid doesn't separate
+# from the boundary
+# Note: The viscosity will be increased by the surface tension model hence we can reduce the artificial viscosity value.
+# Note: The surface tension model leds to an increase in compressibility of the fluid
+#       which needs to be rectified by an increase of the `sound_speed`.
+# Note: The Wendland Kernels don't work very well here since the SurfaceTensionAkinci
+#       model is optimized for smoothing_length=4r which is a too small smoothing_length for the Wendland Kernel functions.
```

Comment:
Because the compact_support has a factor of 2?

Reply:
Particle spacing might be more than 2*r in some cases.

Reply:
So it's not optimized for a smoothing length of 4r, but a compact support of 4r?
> Particle spacing might be more than 2*r in some cases.

The initial particle spacing is per definition equal to 2r, isn't it?

Reply:
Then I suggest:
```
# Note: The Wendland Kernels don't work very well here since the `SurfaceTensionAkinci`
#       model is optimized for a compact support of `2 * particle_spacing`, which would result
#       in a too small `smoothing_length` for the Wendland Kernel functions.
```